plugins/modules/keycloak_userprofile.py

```diff
@@ -436,7 +436,7 @@ def camel_recursive(data):
         # Apply camelCase conversion to each item in the list
         return [camel_recursive(item) for item in data]
     else:
-        # Return the data as is if it's not a dict or list
+        # Return the data as is if it is not a dict or list
```

Comment:
nit picking to improve readability:
```suggestion
        # Return the data as-is if it is not a dict or list
```